plutus-playground/deployment/nixops/default-machine.nix

```diff
@@ -0,0 +1,69 @@
+{ machines, stdOverlays, ... }: node: pkgs:
+{
+      nixpkgs.overlays = stdOverlays;
+      nix = {
```

Comment:
Seems to have a lot of duplication with the other configuration, can we put some of those bits into modules and import them?

Reply:
no duplication, configuration.nix was just for playing around (I presume that's the duplication you meant)?